challenges-webapp/src/main/webapp/vue-app/challenges-extensions/challengesExtensions.js

```diff
@@ -22,7 +22,11 @@ export function initExtensions() {
       };
     },
     getThumbnail: () => '/challenges/images/challengesAppIcon.png',
-    getSummary: activity => activity && activity.templateParams && activity.templateParams.announcementDescription  || '',
+    getSummary: activity => {
+      const announcementDescription = activity && activity.templateParams && activity.templateParams.announcementDescription  || '';
+      const announcementChallengeId = activity && activity.templateParams && activity.templateParams.announcementChallengeId  || '';
+      return `<a href="${eXo.env.portal.context}/${eXo.env.portal.portalName}/challenges/${ announcementChallengeId }"> ${ announcementDescription } </a> `;
```

Comment:
Could you just send the needed information from here, and add the link tag to the template.